jskit/encoders/utils/tokenizer.py

```diff
@@ -96,8 +96,76 @@ def batchify_join_str(self, batch):
         return contexts_token_ids_list_batch, contexts_input_masks_list_batch, \
             candidates_token_ids_list_batch, candidates_input_masks_list_batch, labels_batch
 
+# this class transforms data in required training format and inference for text and tokens
+
+
+class InferDataset(Dataset):
```

Comment:
I am not sure what's the purpose of this. Was this not needed before for inference?

Reply:
This is removed